### ISSUE TITLE
Close button on "titlebar"

### ISSUE THREAD

**Issue by bu-but** [herbstluftwm/herbstluftwm#1583]
Is it possible to have a little "X" button that can be clicked to close a window? I saw someone using it in a screenshot, but couldn't find any information about it in the documentation. Does that actually exist?

**Comment by t-wissmann** [comment]
Hi, it's not possible. Did the creator of the screenshot maybe patch the sourcecode manually? (Just a guess)

**Comment by bu-but** [comment]
Yeah, it seems like that's the case. Perhaps it's worth considering implementing it? I could give it a try with my limited C++ skills, but I can't promise top-notch code.

**Comment by t-wissmann** [comment]
I'm not opposed to title bar buttons, but it is probably better to start with smaller changes.
Also, my original plan was to have generic configurable title bar buttons, such that everyone can adjust it to their needs (close, minimize, maximize, sticky, etc)

**Comment by bu-but** [comment]
Something like openbox, allowing the user to specify icons (maybe svg files) choosing what buttons they might want?
